Sources/EduIDExpansion/SharedViews/File.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
```

Comment:
This file is named "File.swift", it should be "BasicStackView.swift".

Reply:
I renamed the file